.github/workflows/deploy_docs.yml

```diff
@@ -0,0 +1,42 @@
+name: Build & deploy sphinx document
+
+on:
+  workflow_dispatch:
```

Comment:
Would it be possible to also add a manual trigger for that workflow, by any chance ?

Reply:
I have added a `workflow_dispatch` trigger, so authorized people should be able to run the workflow manually as well.

A manual trigger should appear on actions tab like this:
![image](https://github.com/goodchemistryco/Tangelo/assets/103920010/3f0e87c6-8153-48f0-8fbe-699d64d44e96)